pkg/apis/resource/validation/validation_resourceclaim_test.go

```diff
@@ -416,10 +420,31 @@ func TestValidateClaimStatusUpdate(t *testing.T) {
 				claim.Status.Allocation = &resource.AllocationResult{
 					Devices: resource.DeviceAllocationResult{
 						Results: []resource.DeviceRequestAllocationResult{{
-							Request: badName,
+							Request:     badName,
+							Driver:      goodName,
+							Pool:        goodName,
+							Device:      goodName,
+							AdminAccess: ptr.To(false),
+						}},
+					},
+				}
+				return claim
+			},
+		},
+		"invalid-add-allocation-missing-admin-access": {
+			wantFailures: field.ErrorList{
+				field.Required(field.NewPath("status", "allocation", "devices", "results").Index(0).Child("adminAccess"), ""),
+			},
+			oldClaim: validClaim,
+			update: func(claim *resource.ResourceClaim) *resource.ResourceClaim {
+				claim.Status.Allocation = &resource.AllocationResult{
+					Devices: resource.DeviceAllocationResult{
+						Results: []resource.DeviceRequestAllocationResult{{
+							Request: goodName,
 							Driver:  goodName,
 							Pool:    goodName,
 							Device:  goodName,
+							// AdminAccess: ptr.To(false),
```

Comment:
If you want to leave this in, I'd probably comment on it.

Whenever I see comments I think it was an accident to be left behind. In this case, I guess you wanted to add a comment to make it clear that AdminAccess is not specified.

Reply:
TBH the test name is clear enough without the comment IMO.

Reply:
Changed to:
```
AdminAccess: nil, // Intentionally not set.
```

I want to call out the difference compared to the other case.